### PR TITLE
made example simpler

### PR DESCRIPTION
**Pull request by mehradans92**
@whitead 
https://nbviewer.org/github/ur-whitelab/peptide-dashboard/blob/quick_start/examples/Quick_start.ipynb

### CODE REVIEW

**Comment by mehradans92** [comment]
Sure. Removed logits. @whitead
